packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -216,28 +214,38 @@ export default {
   mounted() {
     this.loadData()
   },
+  asyncComputed: {
+    preview: {
+      async get() {
+        await new Promise(resolve => setTimeout(resolve, 500))
```

Comment:
Could you add a comment that the waiting happens to suppress flickering in the UI and clever fixes are welcome? Fine by me to keep the code like it is for now.